engine/conditions.go

```diff
@@ -116,12 +115,17 @@ type CustomVariable struct {
 }
 
 type Perform struct {
-	Message *string  `yaml:"message,omitempty"`
+	Message Message  `yaml:",inline"`
 	Tag     []string `yaml:"tag,omitempty"`
 }
 
+type Message struct {
+	Text  *string       `yaml:"message,omitempty"`
+	Links []hubapi.Link `yaml:"links,omitempty" json:"links,omitempty"`
```

Comment:
Any reason only links has a json tag?

Reply:
Nope they should both or neither will update!

Reply:
Updated, resolving